docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,62 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user with many people in the address book |list all people in address book |see whose contacts the address book already has
```

Comment:
I would argue that people even with few people in the address book might want to list them. :)